utils/queues/lua/fair_sorted_pause.lua

```diff
@@ -0,0 +1,7 @@
+local activeSetKey = KEYS[1]
+local ownerID = ARGV[1]
+
+local score = redis.call("ZSCORE", activeSetKey, ownerID)
+if score ~= false then
+    redis.call("ZADD", activeSetKey, 1000000 + score % 1000000, ownerID)
+end
```

Comment:
initially I was just doing `ZINCRBY` from the go class.. but this has the advantage of being tolerant to be calling more than once and not getting things into a weird state...